### PR TITLE
Updated the README

### PR DESCRIPTION
**Pull request by kuutsav**
- Updated the README to fix some typos and grammatical errors
- ~~Fixed a broken import~~
- ~~Quite a few pre-commit changes made by black~~

### CODE REVIEW

**Comment by lalitpagaria** [comment]
@kuutsav Thank you for working on it.
I will review and share feedback with you.

**Comment by lalitpagaria** [comment]
Kindly create separate PR for black formatting-related changes, as it would be easier to review.

**Comment by kuutsav** [comment]
> Kindly create separate PR for black formatting-related changes, as it would be easier to review.

Reverted two commits. This PR just contains the updates to the README now.

File: README.md [hunk 1]
```diff
@@ -733,126 +771,130 @@ analyzer = DummyAnalyzer()
...
-    cred_info=ZendeskCredInfo(
-        email="<zendesk_user_email>",
-        password="<zendesk_password>"
-    )
+   # For custom domain refer http://docs.facetoe.com.au/zenpy.html#custom-domains
+   # Mainly you can do this by setting the environment variables:
+   # ZENPY_FORCE_NETLOC
+   # ZENPY_FORCE_SCHEME (default to https)
+   # when set it will force request on:
+   # {scheme}://{netloc}/endpoint
```
Comment (lalitpagaria) [inline README.md:806]: These lines are stale now, I forgot to remove it in https://github.com/obsei/obsei/pull/221

**Review by lalitpagaria — APPROVED** [review]
Thanks, @kuutsav for fixing Readme.
I have one comment, the rest all is fine.

**Comment by kuutsav** [comment]
> Thanks, @kuutsav for fixing Readme. I have one comment, the rest all is fine.

Done!

**Pull request merged**